app/styles/custom-components/_vlc-agenda-items.scss

```diff
@@ -199,3 +192,51 @@ $i: 0;
   opacity: 1;
   color: $napier-green;
 }
+
+.vlc-agenda-items__body {
+  align-items: center;
+  padding: 1.2rem 1.8rem;
```

Comment:
This is best to become `2rem` instead. Here's the design:
![image](https://user-images.githubusercontent.com/1874332/82882517-743bfb00-9f41-11ea-9076-38ed65600007.png)

Here's the current app:
![image](https://user-images.githubusercontent.com/1874332/82882546-7e5df980-9f41-11ea-9e20-7798ea70c61f.png)